BaseProject#1/SOEN341InstagramReplica/SOEN341InstagramReplica/Views/Users/RemoveLink.cshtml

```diff
@@ -0,0 +1,40 @@
+﻿@model SOEN341InstagramReplica.Models.LinkedAccount
```

Comment:
How am I suppose to go to this page?
Also what is the difference between ~/Views/Users/RemoveLink.cshtml and ~/Views/LinkedAccounts/Delete.cshtml ?

Reply:
will deal with

Reply:
removed removelink.cshhtml